lib/presentation/atoms/toggle_button_atom.dart

```diff
@@ -29,7 +29,14 @@ class _ToggleButtonAtomState extends State<ToggleButtonAtom> {
 
   @override
   Widget build(BuildContext context) {
+    final ThemeData themeData = Theme.of(context);
+    final ColorScheme colorScheme = themeData.colorScheme;
     return AnimatedToggleSwitch<bool>.rolling(
+      style: ToggleStyle(
+        backgroundColor: colorScheme.outline,
```

Comment:
Please change the background color to outline only when the toggle is off.

Maybe also the borderColor, test how does it look